floor_generator/lib/value_object/entity.dart

```diff
@@ -45,7 +48,13 @@ class Entity extends Queryable {
 
     final withoutRowidClause = withoutRowid ? ' WITHOUT ROWID' : '';
 
-    return 'CREATE TABLE IF NOT EXISTS `$name` (${databaseDefinition.join(', ')})$withoutRowidClause';
+    if (fts == null) {
+      return 'CREATE TABLE IF NOT EXISTS `$name` (${databaseDefinition.join(', ')})$withoutRowidClause';
+    } else {
+      final ftsType = fts.type ?? '';
+      final ftsTokenizer = fts.tokenizer ?? '';
+      return 'CREATE VIRTUAL TABLE IF NOT EXISTS `$name` USING $ftsType(${databaseDefinition.join(', ')}, tokenize=$ftsTokenizer)$withoutRowidClause';
```

Comment:
❓ Does FTS work without a `rowid`?

```suggestion
      return 'CREATE VIRTUAL TABLE IF NOT EXISTS `$name` USING $ftsType(${databaseDefinition.join(', ')}, tokenize=$ftsTokenizer)';
```

Reply:
ths，next commit will change